src/pybind/mgr/dashboard/module.py

```diff
@@ -252,6 +252,17 @@ class Module(MgrModule, CherryPyConfig):
             "desc": "Push dashboards to Grafana",
             "perm": "w",
         },
+        {
+            'cmd': 'dashboard set-cluster-status '
+                   'name=status,type=CephString',
+            'desc': 'Set the current status o the cluster',
+            'perm': 'w',
+        },
+        {
+            'cmd': 'dashboard get-cluster-status',
+            'desc': 'Get the current status o the cluster',
+            'perm': 'r',
+        },
     ]
```

Comment:
What about adding this to config-key? that would avoid adding new user-facing CLI commands for a purly internal boolean. I'd be :+1: for keeping those things hidden from the user. `ceph config-key set mgr/dashboard/cluster-status INSTALLED`

Reply:
@sebastian-philipp It would be good if cephadm can set this value initially, either to INSTALLED or POST_INSTALLED. 
```
INSTALLED - cephadm bootstrap completed
POST_INSTALLED - cluster creation completed
```
If the user preferred to create the cluster using the cli after cephadm bootstrap is completed and then switch to the dashboard, then I don't think we need to show this page to the user (it'll be misleading). So in that case I think cephadm should set this flag initially and pass it down to the dashboard and the dashboard acts accordingly. If `config-key` makes it easier, I'll make the changes. What do you think?

Reply:
Done.